include/getfilecontents.h

```diff
@@ -21,11 +22,23 @@
   SOFTWARE.
 */
 
+/**
+ * @file getfilecontents.h
+ * @brief Fills the econf_file struct with data from a given file.
+ */
```

Comment:
This is the description for a whole file (even though it only contains one function), so you cannot use an active word like "Fills [...]". An appropriate description could be something like _Contains functions to ..._ .